<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LLNLImportTool</class>
 <widget class="QDockWidget" name="LLNLImportTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LLNL Import Tool</string>
  </property>
  <widget class="QWidget" name="llnl_import_tool_contents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="data">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Instrument</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="instrument">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="instrument_label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Instrument</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="instruments">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>(None)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>TARDIS</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PXRDIP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>BBXRD</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="config">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="1" column="1" colspan="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="config_file_label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string extracomment="Defaults to currently loaded configuration"/>
            </property>
            <property name="text">
             <string>No File Selected</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="load_config">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Load Instrument Config</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="current_config">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Current Configuration</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="select_config">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Load Configuration</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="default_config">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Default</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="raw_image">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Raw Image</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QFrame" name="file_selection">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="load">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Select Image</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="files_label">
            <property name="text">
             <string>(None Selected)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="transform_img">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="transform_label">
            <property name="text">
             <string>Transform Image:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="transforms">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mirror about Vertical</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mirror about Horizontal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Transpose</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rotate 90°</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rotate 180°</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rotate 270°</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="add_transform">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Add Transform</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="association">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Association</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="detector_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Current Detector</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="detectors">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_template">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Add Detector Outline</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="outline_position">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Position Detector Outline</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="trans">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Translate</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">positionOutlineButtons</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="rotate">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Rotate</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">positionOutlineButtons</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignRight">
        <widget class="QDialogButtonBox" name="button_box">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
         <property name="centerButtons">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="outline_appearance">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Edit Detector Outline</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="bbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="bb_label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Bounding Box</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="width_label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="bb_width">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>99</number>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="height_label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="bb_height">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>99</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="style">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="color_label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Color:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="line_color">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="style_label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Line Style:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="line_style">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>solid</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>dotted</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>dashdot</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="size_label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Line Width:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QDoubleSpinBox" name="line_size">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="keyboardTracking">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="finalize">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Finalize Instrument</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="completed_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Completed Detectors:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="completed_dets">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="cancel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>105</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="complete">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Complete and Reload Instrument</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="positionOutlineButtons"/>
 </buttongroups>
</ui>
